snapshots/artificial_intelligence/2023-06-08/yougov_end_of_humanity.pdf.dvc

```diff
@@ -0,0 +1,25 @@
+meta:
+  namespace: artificial_intelligence
+  short_name: yougov_end_of_humanity
+  name: AI and the End of Humanity (YouGov, 2023)
+  version: '2023-06-08'
+  publication_year: 2023
+  publication_date: '2023-04-03'
+  source_name: YouGov
+  source_published_by: YouGov
+  url:
+    https://today.yougov.com/topics/technology/articles-reports/2023/04/14/ai-nuclear-weapons-world-war-humanity-poll?utm_source=80,000%20Hours%20mailing%20list&utm_campaign=3bb3a9cbc0-EMAIL_CAMPAIGN_2023_05_25_07_19&utm_medium=email&utm_term=0_43bc1ae55c-fcdb5a40fb-%5BLIST_EMAIL_ID%5D
```

Comment:
The URL you need is probably just:
https://today.yougov.com/topics/technology/articles-reports/2023/04/14/ai-nuclear-weapons-world-war-humanity-poll